packages/grid/_modules_/grid/utils/keyboardUtils.ts

```diff
@@ -1,11 +1,32 @@
 export const GRID_MULTIPLE_SELECTION_KEYS = ['Meta', 'Control', 'Shift'];
 export const isMultipleKey = (key: string): boolean =>
   GRID_MULTIPLE_SELECTION_KEYS.indexOf(key) > -1;
+
+export const GRID_CELL_ENTER_EDIT_MODE_KEYS = ['Enter', 'Delete', 'Backspace'];
+export const isCellEnterEditModeKeys = (key: string): boolean =>
+  GRID_CELL_ENTER_EDIT_MODE_KEYS.indexOf(key) > -1;
+
+export const GRID_CELL_EXIT_EDIT_MODE_KEYS = ['Enter', 'Escape', 'Tab'];
+export const isCellExitEditModeKeys = (key: string): boolean =>
+  GRID_CELL_EXIT_EDIT_MODE_KEYS.indexOf(key) > -1;
+
+export const GRID_CELL_EDIT_COMMIT_KEYS = ['Enter', 'Tab'];
+export const isCellEditCommitKeys = (key: string): boolean =>
+  GRID_CELL_EDIT_COMMIT_KEYS.indexOf(key) > -1;
+
 export const isTabKey = (key: string): boolean => key === 'Tab';
```

Comment:
As a side note. This looks overengineered. It used to be relevant when we were testing .keyCode === 9. But now, what you would rather read?

```jsx
if (event.key === 'Tab') {
```

or

```jsx
if (isTabKey(event)) {
``` 

I personally think it's the same, so we might as well remove one indirection and consolidate on a single approach (we already use .key === 'Tab' elsewhere).

Reply:
I'd rather keep the function and avoid magic strings.
- Less error prone, ie someone could easily do event.key ==='tab' or === 'Tab '
- We centralise the way we check for the tab key
- Easier to refactor
- DRY

Reply:
I personally think that these 4 reasons are not valid:

1. Only applies if when coding without testing (ci or manually), which shouldn't happen
2. Currently, removing isTabKey would increase consistency. 
3. Why would we ever need to refactor it? 
4. DRY isn't a goal, it's a mean to an end. So it shouldn't be used as an argument.

But no strong preferences, I wanted to raise the inconsistency and the potential improvement potential.